components/list/link/src/index.js

```diff
@@ -2,26 +2,43 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
 
-import LinkBasic from '@schibstedspain/sui-link-basic'
+import LinkBasic from '@s-ui/react-link-basic'
+
+function ListLinkItem({displayInline, item, useReactRouterLinks}) {
```

Comment:
Extracted the method to a component in order to perform better instead a render method.